packages/publish-spec-to-team/src/index.ts

```diff
@@ -0,0 +1,106 @@
+import { Command } from "commander";
+import fetch, { fileFrom, FormData } from "node-fetch";
+import path from "path";
+import esbuild from "esbuild";
+import { createFile, exec, validateSpecName } from "./utils";
+import { API_BASE, cleanTempDir, tempDir } from "./constants";
+// @ts-ignore package.json is always in '../' as long as we build src/ to lib/
+import packageJSON from "../package.json";
+import { BuildError, GenericErrorEnum, PublishError, ValidationError } from "./errors";
+
+export interface RunOptions {
+  name: string;
+  token: string;
+  specPath?: string;
+  binaryPath?: string;
+  subcommandName?: string;
+  framework?: string;
+}
+
+const program = new Command()
+  .name(packageJSON.name)
+  .version(packageJSON.version)
+  .description("Publish a spec to fig teams");
+
+program
+  .requiredOption("-n, --name <string>", "Set the name of the published spec")
+  .requiredOption("-t, --token <string>", "A fig token")
+  .option("-p, --spec-path <path>", "The local path of the spec to publish")
```

Comment:
Should `name` really be required? Doesn't this make sense to default to the `CLI.ts` or `./bin/CLI` and optionally allow the user to change the name?

Reply:
👍